### PR TITLE
vagrant: update README and add info about vboxusers group

### PR DESCRIPTION
**Pull request by cgundogan**
leave a hint about `vboxusers` group.

This PR updates obsolete information in the vagrant READM and replaces `RIOT-OS` with the preferred version: `RIOT`.

### CODE REVIEW

**Comment by thomaseichinger** [comment]
Could we mention that this is true for linux?

**Comment by cgundogan** [comment]
@thomaseichinger ofcourse! I amened a small hint to the subtitle (for Linux Users)

**Comment by smlng** [comment]
@cgundogan I nearly ACK this one , but then I saw that some parts are outdated:

* its not Ubuntu _Wiley_ but _Xenial_ , and most users will know it by its number 16.04, add that too
* isn't the bootstrap / provisioning script depcrecated, at least as soon as we provide our own vagrant box, 

on the other hand, you'll have to update the Vagrantfile anyway then

**Comment by cgundogan** [comment]
@smlng I addressed your comments, are you happy now? (:

File: dist/tools/vagrant/README.md [hunk 1]
```diff
@@ -16,7 +16,7 @@ Make sure your system satisfies the latest version of all following dependencies
 * [Vagrant](https://www.vagrantup.com/downloads.html)
```
Comment (smlng) [inline dist/tools/vagrant/README.md]: Typo: /Extnension/Extension/

File: dist/tools/vagrant/README.md [hunk 2]
```diff
@@ -42,15 +42,19 @@ vagrant provision
...
 on the host system. All changes made will be mirrored from the host system to the guest system
 and vice versa.
 
-This feature allows you to conveniently develop code for RIOT-OS with your preferred IDE on
-your host system and use the VM for compiling, flashing devices and running the native port of RIOT-OS.
+This feature allows you to conveniently develop code for RIOT with your preferred IDE on
+your host system and use the VM for compiling, flashing devices and running the native port of RIOT.
 
 ## Additional Information
 For new boards it is necessary to add new udev rules to `dist/tools/vagrant/udev_rules` and
```
Comment (smlng) [inline dist/tools/vagrant/README.md]: isn't `udev` Linux specific as well?

File: dist/tools/vagrant/README.md [hunk 3]
```diff
@@ -42,15 +42,19 @@ vagrant provision
...
 additional usb filters to the Vagrantfile so that VirtualBox is able to capture the devices.
 The needed `vendor id` and `product id` can be obtained by running `vboxmanage list usbhost`.
+
+### Virtualbox (for Linux Users)
```
Comment (smlng) [inline dist/tools/vagrant/README.md]: I don't like standalone subsubsection, their should be at least 2 or none. Improvement: change structure below _Additional Information_ into a item/bullet list?

**Comment by cgundogan** [comment]
@smlng addressed your comments

**Comment by smlng** [comment]
@cgundogan I'm pleased 😄 

I'll hit merge tomorrow, if no other comments or objections until then.

**Pull request merged**